tests/sql/helpers.test.js

```diff
@@ -0,0 +1,59 @@
+const { sqlToCamelCased, aggregateIntoJson } = require('../../src/sql/helpers');
+
+describe('sqlToCamelCased', () => {
+  it('Works correctly', () => {
+    const experimentFields = [
+      'id',
+      'name',
+      'description',
+      'samples_order',
+      'notify_by_email',
+      'created_at',
+      'updated_at',
+    ];
+
+    expect(sqlToCamelCased(experimentFields)).toEqual([
+      'id as id',
+      'name as name',
+      'description as description',
+      'samples_order as samplesOrder',
+      'notify_by_email as notifyByEmail',
+      'created_at as createdAt',
+      'updated_at as updatedAt',
+    ]);
+  });
+});
+
+describe('aggregateIntoJson', () => {
+  it('Works correctly', async () => {
```

Comment:
what other cases can we test for in here?

Reply:
It doesn't have any ifs or error handling so I didn't really see the need to add another test